heroic_base/settings.py

```diff
@@ -21,14 +21,14 @@
 # See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.getenv('DJANGO_SECRET_KEY')
+SECRET_KEY = os.getenv('DJANGO_SECRET_KEY', 'asdfwea234asdf3wa')
```

Comment:
This is a dummy key I hope?

Reply:
Yes, django just requires you to have some value when running, so I put something in so running in dev works without overrides. We use something different in prod.